Author: Riku Voipio <riku.voipio@linaro.org>
Description: fuse_kernel.h: clean includes
 Use <linux/types.h> for linux and define types used for other operating systems
 using <stdint.h> types.
Origin: backport, http://sourceforge.net/p/fuse/fuse/ci/914871b

Index: fuse-2.9.2/include/fuse_kernel.h
===================================================================
--- fuse-2.9.2.orig/include/fuse_kernel.h	2013-10-11 16:30:25.000000000 +1100
+++ fuse-2.9.2/include/fuse_kernel.h	2013-10-11 16:31:43.618741227 +1100
@@ -88,12 +88,16 @@
 #ifndef _LINUX_FUSE_H
 #define _LINUX_FUSE_H
 
-#include <sys/types.h>
+#ifdef __linux__
+#include <linux/types.h>
+#else
+#include <stdint.h>
 #define __u64 uint64_t
 #define __s64 int64_t
 #define __u32 uint32_t
 #define __s32 int32_t
 #define __u16 uint16_t
+#endif
 
 /*
  * Version negotiation:
